iyaremye-faustin/G11-Backend
# user-roles-settings

### User Roles Settings
Implement roles for users of the application 
### What does the PR do?
It allows Admin user to change/assign other users roles as needed 
### How to test the PR ?
Once the server is running and all packages are installed, head over to {serverUrl}/api/documentation and use Roles tag routes to test it. 
Note that you should be logged in as Administrator to perform this.
